tools/src/main/scala/scala/scalanative/linker/ClassPath.scala

```diff
@@ -46,18 +46,24 @@ object ClassPath {
     def contains(name: Global) =
       files.contains(name.top)
 
+    private def makeBufferName(directory: VirtualDirectory, file: Path) =
+      directory.uri.toString + file
```

Comment:
Concatenating the string representation of a URI with the string representation of a Path does not seem right. Can we make that more semantic?

Reply:
Done